invenio_rdm_records/records/dumpers.py

```diff
@@ -59,3 +65,46 @@ def load(self, data, record_cls):
         # `None` covers the cases where exceptions were raised in _dump
         parent_data.pop(f"{self.key}_start", None)
         parent_data.pop(f"{self.key}_end", None)
+
+
+class LocationsDumper(ElasticsearchDumperExt):
```

Comment:
Minor: I would create a `dumper` folder and then split this dumpers by "functionality" (i.e. `edtf.py`, `location.py`). It might be clearer.

Reply:
Yep. This would certainly be a sensible idea!

Reply:
Now done :-)